integrationtests/test_i18n.py

```diff
@@ -2,6 +2,39 @@
 from integrationtests.helpers import accounts
 
 
+_locale_names = {"en": "English", "el": "Ελληνικά"}
+
+
+class LocaleSwitcherIntegrationTest(WorkbenchBase):
```

Comment:
Don't create base test classes. Nobody should ever import from this file, so there's no possibility this code will be reused. Just put the helpers in the test class itself.